### DISCUSSION TITLE
Adventure mode configuration.

### DISCUSSION THREAD

**Discussion by agoniaprivat** [PurpurMC/Purpur#880] [Feature Requests]
We can add NBT tags to allow specifik blocks to be broken and placed in adventure mode. But could we get a config to allow specifik blocks to be placed and broken globally over the server?

**Comment by YouHaveTrouble** [comment]
I believe client has to have it spelled out on the item nbt, otherwise it won't even send interact packet with block, so no way to tell if they're trying to destroy it.

**Reply by agoniaprivat** [reply]
You can have a hidden NBT tag for This. There is no way the block could get this tag added when rightclick placing the block?
